### ISSUE TITLE
Support for XL-3528RGBW-WS2812B?

### ISSUE THREAD

**Issue by DatanoiseTV** [JSchaenzle/ESP32-NeoPixel-WS2812-RMT#17]
I've tried your code on an ESP32-S3 yesteday and couldn't get the LED to work (no sign of life at all).
Here is a datasheet, if that helps: https://datasheet.lcsc.com/lcsc/2211021800_XINGLIGHT-XL-3528RGBW-WS2812B_C2890364.pdf

**Comment by davidhbrown** [comment]
Datanoise, these are 5V parts. Are you providing both 5V power and a 5V signal? The ESP32, of course, is only 3.3V unlike many (older) Arduinos. I'm using a pair of the similar XL-5050RGBC-WS2812B on a board I had JLCPCB make for me (part C2843785) and I used TI's TXS0102D (C53434) to translate the ESP32's 3.3V logic to the 5V levels needed. Wiring it up is really easy if you already have 5V in your design... just need a couple of 100nF capacitors (C14663). It overshoots a tiny bit but works fine.

**Comment by DatanoiseTV** [comment]
I am aware of that :) I am using a MC74VHC1G08DBVT1G as "level shifter", which works great on other boards with RP2040 etc.
Right now on another project, but I will try to find the right timings.

**Comment by davidhbrown** [comment]
Yeah, it does seem like the timings for the T0L and T1L are much longer in that datasheet than for the part I'm using.
